plugin/trino-postgresql/src/test/java/io/trino/plugin/postgresql/PostgreSqlQueryRunner.java

```diff
@@ -58,6 +58,7 @@ public static DistributedQueryRunner createPostgreSqlQueryRunner(
             connectorProperties.putIfAbsent("connection-password", server.getPassword());
             connectorProperties.putIfAbsent("allow-drop-table", "true");
             connectorProperties.putIfAbsent("postgresql.include-system-tables", "true");
+            connectorProperties.putIfAbsent("postgresql.experimental.enable-string-pushdown-with-collate", "true");
```

Comment:
Since you've now added a session property I suggest we add dedicated test method to TestPostgresqlConnectorTest which sets the session property and then verifies the result.

My reasoning is that tests should cover as much of "default" behaviour as possible too.